.github/workflows/tag-release.yml

```diff
@@ -216,3 +216,75 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
       - run: yarn release
+
+  # Deployment steps taken from https://github.com/colinwilson/static-site-to-vercel/blob/master/.github/workflows/deploy-preview.yml
+  repl_build:
```

Comment:
How come this isn't in the nightly workflow as well?

Reply:
probably don't want to deploy to production on every nightly, just when we do stable releases.